src/base/neural/owl_neural_neuron.ml

```diff
@@ -2043,14 +2043,16 @@ module Make
       mkpri l |> Array.map copy_primal' |> update l';
       l'
 
+    (* Reference: https://arxiv.org/abs/1502.03167.
+       Implementaton in Keras: https://bit.ly/2vBsvgI.*)
     let run x l =
-      let a = F (1. /. float_of_int (shape x).(l.axis)) in
       if l.training = true then (
+        let a = F (float_of_int ((numel x) / (shape x).(l.axis))) in
```

Comment:
Are you sure this is correct? Shouldn't it be `F ((float_of_int (numel x)) / (float_of_int (shape x).(l.axis)))` ... please double check the reference.

And the link to the original paper is still missing.